packages/server/src/fhir/routes.ts

```diff
@@ -207,6 +207,7 @@ function initInternalFhirRouter(): FhirRouter {
 
   // Bot $deploy operation
   router.add('POST', '/Bot/:id/$deploy', deployHandler);
+  router.add('GET', '/Bot/:id/$deploy', deployHandler);
```

Comment:
Hm, we try to avoid using `GET` requests for non-idempotent actions.  I suppose you could argue that repeatedly deploying is somewhat idempotent, but imho it kinda goes against the spirit.

It seems like it should be possible to just make `code` optional within the post handler, yeah?